src/data_fetch_scripts/fetch_electricity_data.py

```diff
@@ -0,0 +1,30 @@
+"""
+This module fetches weather data from the NationalGrid API
+"""
+import os
+import sys
+
+# Add the parent directory to the Python path to import WeatherData class
+CURRENT_DIR = os.path.normpath(os.path.dirname(os.path.abspath(__file__)))
+parent_dir = os.path.normpath(os.path.dirname(CURRENT_DIR))
+sys.path.append(parent_dir)
+
+from classes.electricity_data_api import ElectricityData
+
+
+def main(current_dir: str) -> None:
+    """
+    This function fetches data from the National Grid API and stores it in the data
+    directory
+
+    Args:
+        * current_dir (str): File's current directory
+    """
+    data_dir = os.path.join(current_dir, os.path.normpath("../../data/electricity_demand_data"))
```

Comment:
It would a good idea to define paths in a .json file as part of the configuration of the project. Implement this in another branch